client/src/modules/list/model/actions.js

```diff
@@ -517,3 +526,21 @@ export const changeType = (listId, listName, type) => dispatch =>
         data: listName
       });
     });
+
+export const leaveList = (listId, userId, cohortId, userName) => dispatch =>
+  patchData(`/api/lists/${listId}/leave-list`)
+    .then(() => {
+      dispatch(leaveListSuccess(listId, userId));
+      createNotificationWithTimeout(dispatch, NotificationType.SUCCESS, {
+        notificationId: 'list.actions.leave-list',
+        data: userName
+      });
+      history.replace(`/${cohortId ? 'cohorts' : 'dashboard'}`);
```

Comment:
hmm, why if you leave the cohort's list you should be redirected to all cohorts view? To the dashboard or this specific cohort view - what do you think?